src/crawlee/configuration.py

```diff
@@ -207,7 +217,7 @@ class Configuration(BaseSettings):
             )
         ),
     ] = None
-    """This setting is currently unused. For more details, see https://github.com/apify/crawlee-python/issues/670."""
+    """This setting is currently unused."""
```

Comment:
I think it's safe to just drop this and never look back.

Reply:
I wrote it down to this issue: https://github.com/apify/crawlee-python/issues/906

Reply:
I mean, I don't think there's any harm in removing an option that doesn't do anything, but suit yourself :slightly_smiling_face: